mbed_host_tests/host_tests_plugins/host_test_plugins.py

```diff
@@ -195,13 +193,10 @@ def check_serial_port_ready(self, serial_port, target_id=None, timeout=60):
                             new_serial_port = mbeds_by_tid[target_id]['serial_port']
                             break
                 sleep(timeout_step)
+        else:
+            new_serial_port = serial_port
 
-            if new_serial_port != serial_port:
-                # Serial port changed, update to new serial port from mbed-ls
-                self.print_plugin_info("Serial port for tid='%s' changed from '%s' to '%s'..."% (target_id, serial_port, new_serial_port))
```

Comment:
Seems like this could still be useful debug info, mind moving this logic into the above loop?